app/templates/archive.html

```diff
@@ -0,0 +1,26 @@
+{% extends "./partials/notification/base.html" %}
+{% block page_name %}Archive{% endblock page_name %}
+
+{% block description %}
+    <div>
+        <div class="title">Archived Notifications</div>
+        <p class="s-paragraph">
+            In this page you can view all of your archived notifications.<br/>
+            Any notification you have <b>marked as read</b> or <b>declined</b>, you will see here.<br/>
+            You can use the <button class="notification-btn btn-accept"><ion-icon class="icon" name="checkmark-outline"></ion-icon></button>
+            button to accept an invitation that you already declined.
+        </p>
+    </div>
+{% endblock description %}
+
+{% block link %}
+    <div><h5><a href="{{ url_for('view_notifications') }}">New notifications</a></h5></div>
```

Comment:
Why h5? If because of the size - use CSS instead.
If it should be a title - choose the appropriate title according to the hierarchy. (Is this element have 4 titles above it?)